### PR TITLE
BZ-1843158: Added link to removing credentials for admin-level

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.7+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1843158
Pending QE Approval: 
Preview Links: 

- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-account.html#installation-aws-iam-user_installing-aws-account [Second bullet under Additional Resources]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-default.html#next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc.html#next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-private.html#next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-government-region.html#next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installing-aws-user-infra-next-steps [4th bullet]
- https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installing-restricted-networks-aws-next-steps [5th bullet]

### CODE REVIEW

File: installing/installing_aws/installing-aws-account.adoc [hunk 1]
```diff
@@ -21,6 +21,7 @@ include::modules/installation-aws-iam-user.adoc[leveloffset=+1]
 .Additional resources
 
 * See xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[Manually creating IAM for AWS] for steps to set the Cloud Credential Operator (CCO) to manual mode prior to installation. Use this mode in environments where the cloud identity and access management (IAM) APIs are not reachable, or if you prefer not to store an administrator-level credential secret in the cluster `kube-system` project.
+* See xref:../../authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc#manually-removing-cloud-creds_cco-mode-mint[Removing cloud provider credentials] to remove the administrator-level credential secret from the `kube-system` namespace in the cluster. The administrator-level credential is not required unless something that requires an administrator-level credential needs to be changed, for instance during an upgrade.
```
Comment (kalexand-rh) [inline installing/installing_aws/installing-aws-account.adoc:24]: If you changed this text in the 4.5 version, please update this text, too.

Comment (yunjiang29) [reply installing/installing_aws/installing-aws-account.adoc]: @sagidlow please keep text consistent with version 4.5 and 4.6

**Comment by yunjiang29** [comment]
@lwan-wanglin PTAL, thanks

**Comment by lwan-wanglin** [comment]
@sagidlow,  as @yunjiang29's suggest, it would be better to keep text consistent in 4.7+ with version 4.5 and 4.6, others look good to me, thanks.

in 4.7(https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-mint.html#manually-removing-cloud-creds_cco-mode-mint) it is still:
`The administrator-level credential is required only during changes that require its elevated permissions, such as upgrades.`

in 4.5 and 4.6, you have updated to:
`The administrator-level credential is required only during changes that require its elevated permissions, such as upgrades.`

**Comment by lwan-wanglin** [comment]
@sagidlow,  as @yunjiang29's suggest, it would be better to keep text consistent in 4.7+ with version 4.5 and 4.6, others look good to me, thanks.

in 4.7(https://deploy-preview-32144--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-mint.html#manually-removing-cloud-creds_cco-mode-mint) it is still:
`The administrator-level credential is required only during changes that require its elevated permissions, such as upgrades.`

in 4.5 and 4.6, you have updated to:
`The administrator-level credential is required only during changes that require its elevated permissions, such as upgrades.`

**Pull request merged**